contrib/systemd/etcd2-backup/README.md

```diff
@@ -0,0 +1,249 @@
+# etcd2-backup
+
+Remote backup and multi-node restore services for etcd2 clusters.
+
+## Terminology
+
+**founding member** : The node which is the first member of the new recovered cluster. It is this node's rclone backup data (only) that will be used to restore the cluster. The rest of the nodes will join the cluster with no data, and simply catch up with the **founding member**.
```

Comment:
add a note to say it only works well for small data set? For large data set, they'd better just copy the data dir to all members to avoid snapshot sending inside etcd at least for now.